projects/demo/src/modules/components/confirm/index.ts

```diff
@@ -29,6 +30,7 @@ export default class ExampleComponent implements TuiConfirmData {
     private readonly dialogs = inject(TuiDialogService);
     private readonly alerts = inject(TuiAlertService);
 
+    protected readonly docRoutes = DemoRoute;
```

Comment:
why not just `protected readonly routes = DemoRoute;`?

Reply:
No problems, I'll rename it

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/7642/commits/883751d9ff2f3ed444f7f1dc49498204c0b476ca